src/components/reorderer/css/Reorderer.css

```diff
@@ -19,55 +21,56 @@
 
 /* automatically applied as the mouse hovers over re-orderable item */
 .fl-reorderer-movable-hover {
-    background-color: lightyellow; /* !important;;*/
+    background-color: lightyellow; /* !important;; */
```

Comment:
This seems to be an odd comment. I wonder if we should just remove it.

Reply:
I think so. I'm not sure why it's there.